ckanext/unaids/plugin.py

```diff
@@ -255,7 +255,8 @@ def _giftless_upload(context, resource, current=None):
             })
 
 
-def _update_resource_last_modified_date(resource, current={}):
+def _update_resource_last_modified_date(resource, current=None):
+    current = current or {}
```

Comment:
That's an anti pattern. Current might evaluate to false in many other cases then just `None` leading to hard to debug errors.
I suggest using the standard:
```
if current is None:
  current = {}
```

One example discussion can be found here in the comments: https://stackoverflow.com/a/13710667/3432120